partition/roles/metal-core/files/metal-core.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Delays startup of metal-core until SONiC has started properly
+
+[Timer]
+OnUnitActiveSec=0 sec
+OnBootSec=3min 30 sec
```

Comment:
`metal-core` should handle it on its own instead of introducing some random interval.